include/api/wasmedge/wasmedge.h

```diff
@@ -2379,6 +2379,16 @@ WASMEDGE_CAPI_EXPORT extern void WasmEdge_ImportObjectInitWASI(
     const uint32_t ArgLen, const char *const *Envs, const uint32_t EnvLen,
     const char *const *Preopens, const uint32_t PreopenLen);
 
+/// Get the WASI exit code.
+///
+/// This function will return the exit code from the WASI host module.
+///
+/// \param Cxt the WasmEdge_ImportObjectContext of WASI import object.
+///
+/// \returns the exit code. Return `EXIT_FAILURE` if error occurs.
```

Comment:
According to your implementation, it can return `EXIT_SUCCESS`, `EXIT_FAILURE`, or any possible `EXITCODE`.